molecularnodes/entities/molecule/base.py

```diff
@@ -65,6 +64,12 @@ def __init__(self, file_path: Union[str, Path, io.BytesIO]):
         self._frames_collection: str | None
         self._entity_type = EntityType.MOLECULE
 
+
+    @classmethod
```

Comment:
Preferred entry point is now a class method instead of over loading the __init__/ Much cleaner